app/controllers/users_controller.rb

```diff
@@ -15,6 +16,16 @@ def update
     end
   end
 
+  def delete_card
+    customer = Stripe::Customer.retrieve(params[:customer_id])
+    customer.sources.retrieve(params[:card_id]).delete
```

Comment:
Yeah, it might be used by stripe for other record keeping maybe? Check the Stripe docs to confirm if we need to delete from Stripe too?